### ISSUE TITLE
Cant tei use multiple gpus to serve in parallel? i assign mutliple gpus for a tei docker service, but it only works on a gpu

### ISSUE THREAD

**Issue by t6am3** [huggingface/text-embeddings-inference#206]
### System Info

ghcr.io/huggingface/text-embeddings-inference:latest

### Information

- [X] Docker
- [ ] The CLI directly

### Tasks

- [X] An officially supported command
- [ ] My own modifications

### Reproduction

docker run -d --name tei_rerank \
           -v ./models/rerank/BAAI_bge_reranker_large:/reranker_model:ro \
           --gpus '"device=2,3,4,5"' \
           -p 39672:80 \
           ghcr.io/huggingface/text-embeddings-inference:latest \
           --model-id /reranker_model \
           --revision None \
           --dtype float32 \

### Expected behavior

After multiple gpus passed, this service could process my requests in parallel

**Comment by OlivierDehaene** [comment]
Duplicate of #87
